e2e-tests/cypress/support/appellant-submission-decision-letter/confirmDecisionLetterRejectedBecause.js

```diff
@@ -1,6 +1,6 @@
 module.exports = (errorMessage) => {
   // confirm we are in the right place
-  cy.url().should('include', '/appellant-submission/upload-decision');
+  cy.url().should('include', '/appeal-householder-decision/upload-decision');
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks